allennlp/models/model.py

```diff
@@ -49,13 +49,13 @@ def __init__(self,
         self.vocab = vocab
         self._regularizer = regularizer
 
-    def get_regularization_penalty(self) -> Optional[torch.Tensor]:
+    def get_regularization_penalty(self) -> torch.autograd.Variable:
         """
-        Get the regularization penalty for the model. Will be ``None``
-        if there's no regularizer.
+        Computes the regularization penalty for the model.
+        Returns a zero tensor if the model was not configured to use regularization.
         """
         if self._regularizer is None:
-            return None
+            return torch.autograd.Variable(torch.Tensor([0.]))
```

Comment:
This breaks if the model is on the GPU. I think you can just return `0.0` instead?

Reply:
done